lib/inputs/car_input.dart

```diff
@@ -13,13 +13,13 @@ class CarFormInput {
 int calculateEmission(String type, double distance) {
   switch (type) {
     case "Medium":
-      return (0.297 * distance).floor(); // emissions (kg/km) * distance
+      return (0.297 * distance).floor() * 1000; // emissions (kg/km) * distance * 1000 (gram conversion)
```

Comment:
You would want to `*1000` before the `floor()` because `floor()` would convert it to an inaccurate number.